DOCUMENTATION.md

```diff
@@ -181,6 +181,27 @@ email_reply_to_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 
 If you omit this argument, the client will set your default email reply-to address for the notification.
 
+### Attach a document
```

Comment:
Might be worth checking with @antimega about "attach" vs "upload". Email attachments are a specific thing and that's explicitly not what's going to happen in this case.